CODEOWNERS

```diff
@@ -45,6 +45,7 @@
 /src/lib/daemon_rpcs/ @wu-s-john @bkase @schmavery
 /src/lib/debug_assert/ @nholland94 @bkase @cmr
 /src/lib/dummy_values/ @bkase @imeckler @mrmr1993
+/src/lib/exit_handlers/ @psteckler @mrmr1993
```

Comment:
@psteckler missed this update earlier, but I think this would be a good candidate for leaving unowned, at least while it's only a single function.